apps/condo/migrations/20221222134302-0204_userexternalidentityhistoryrecord_and_more.js

```diff
@@ -0,0 +1,52 @@
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Create model userexternalidentityhistoryrecord
+--
+CREATE TABLE "UserExternalIdentityHistoryRecord" ("user" uuid NULL, "identityId" text NULL, "identityType" text NULL, "meta" jsonb NULL, "id" uuid NOT NULL PRIMARY KEY, "v" integer NULL, "createdAt" timestamp with time zone NULL, "updatedAt" timestamp with time zone NULL, "createdBy" uuid NULL, "updatedBy" uuid NULL, "deletedAt" timestamp with time zone NULL, "newId" jsonb NULL, "dv" integer NULL, "sender" jsonb NULL, "history_date" timestamp with time zone NOT NULL, "history_action" varchar(50) NOT NULL, "history_id" uuid NOT NULL);
+--
+-- Create model userexternalidentity
+--
+CREATE TABLE "UserExternalIdentity" ("identityId" text NOT NULL, "identityType" varchar(50) NOT NULL, "meta" jsonb NULL, "id" uuid NOT NULL PRIMARY KEY, "v" integer NOT NULL, "createdAt" timestamp with time zone NULL, "updatedAt" timestamp with time zone NULL, "deletedAt" timestamp with time zone NULL, "newId" uuid NULL, "dv" integer NOT NULL, "sender" jsonb NOT NULL, "createdBy" uuid NULL, "updatedBy" uuid NULL, "user" uuid NOT NULL);
```

Comment:
what about the data migration?

Reply:
Data migration are done in separate script. There are some business logic behind data migration